wfe/wfe.go

```diff
@@ -585,16 +578,13 @@ func (wfe *WebFrontEndImpl) verifyPOST(ctx context.Context, logEvent *requestEve
 	err = json.Unmarshal([]byte(payload), &parsedRequest)
 	if err != nil {
 		wfe.stats.Inc("Errors.UnparseableJWSPayload", 1)
-		logEvent.AddError("unable to JSON parse resource from JWS payload: %s", err)
```

Comment:
Is this a place where we're going to lose information? The actual parse error might be interesting to know and I don't think it will be preserved anywhere without this `AddError`.

Reply:
In my opinion it's not worth logging the JSON parse error here, since there are a lot of ways users can produce unparseable JSON, but we're not interested in debugging them further than "it was broken". Also, in theory we can repro the error by pulling out the payload from the logs.

Reply:
> Also, in theory we can repro the error by pulling out the payload from the logs.

I thought we weren't logging full payloads anymore.

> In my opinion it's not worth logging the JSON parse error here

SGTM.

Reply:
We're logging the "payload" part, after base64 decoding, as part of WFE logs.

Reply:
Ahhh right. I was thinking of POST bodies not JWS payloads. :+1: